### PR TITLE
100. Which is default user in Ec2 Linux-red hat

### PR DESCRIPTION
**Pull request by chandolakul**
100. Which is default user in Ec2 Linux-red hat/amazon.

### CODE REVIEW

File: aws/aws-quiz.md [hunk 1]
```diff
@@ -227,7 +227,7 @@
 #### Q29. A new developer has been added to the team and you have been asked to provide access to the organization's AWS account. What is the best practice for granting access?
 
 - [ ] Give the new developer the IAM login that is assigned to the development team. This IAM user should already include all of the policies that a developer would need.
-- [ ] Create a IAM user for the new developer. Manually assign policies to the new IAM user account.
+- [ ] Create a IAM user for the new developer. Manually assign policies to the new IAM user acunt.
```
Comment (auchynnikau) [inline aws/aws-quiz.md:230]: The initial word was correct, we don't need to change it. Please, fix this typo.

```suggestion
- [ ] Create a IAM user for the new developer. Manually assign policies to the new IAM user account.
```

Comment (auchynnikau) [inline aws/aws-quiz.md:230]: The initial word was correct, we don't need to change it. Please, fix this typo.

```suggestion
- [ ] Create a IAM user for the new developer. Manually assign policies to the new IAM user account.
```

**Review by auchynnikau — CHANGES_REQUESTED** [review]

**Review by Ebazhanov — CHANGES_REQUESTED** [review]
@chandolakul thanks for your PR, pls review comments

**Comment by chandolakul** [comment]
@Ebazhanov Resolved Conflicts, Please approve the pull request and Kindly add the hacktoberfest-accepted label in this PR.

**Comment by Ebazhanov** [comment]
> @Ebazhanov Resolved Conflicts, Please approve the pull request and Kindly add the hacktoberfest-accepted label in this PR.

what about an other comments, could you pls also fix typos, thanks

**Review by Ebazhanov — APPROVED** [review]
LGTM, 👍

**Comment by chandolakul** [comment]
@Ebazhanov  --- Typo error fixed now..

**Pull request merged**